File: Genral_All_Controls/StandaloneExpressionEditorForm/StandaloneExpressionEditorFormCS/StandaloneExpressionEditorForm.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;
using Telerik.WinControls.UI;

namespace StandaloneRadExpressionEdtiorForm
{
    public partial class StandaloneExpressionEditorForm : RadExpressionEditorForm
    {
        private static RadGridView hiddenGrid;
        private static GridViewDataColumn dataColumn;
        
        static StandaloneExpressionEditorForm()
        {
            hiddenGrid = new RadGridView();
            dataColumn = new GridViewTextBoxColumn();
            hiddenGrid.Columns.Add(dataColumn);
            hiddenGrid.Rows.AddNew();
        }

        public StandaloneExpressionEditorForm() 
            : base(dataColumn)
        {
        }

        public string ReturnValue
        {
            get
            {
                return hiddenGrid.Rows[0].Cells[0].Value.ToString();
            }
        }


        public DialogResult ShowDialog(object initialValue)
        {
            this.Expression = initialValue.ToString();
            return base.ShowDialog();
        }
    }
}

File: Genral_All_Controls/StandaloneExpressionEditorForm/StandaloneExpressionEditorFormVB/StandaloneExpressionEditorForm.vb
Imports Telerik.WinControls.UI

Partial Public Class StandaloneExpressionEditorForm
    Inherits RadExpressionEditorForm
    Private Shared hiddenGrid As RadGridView
    Private Shared dataColumn As GridViewDataColumn

    Shared Sub New()
        hiddenGrid = New RadGridView()
        dataColumn = New GridViewTextBoxColumn()
        hiddenGrid.Columns.Add(dataColumn)
        hiddenGrid.Rows.AddNew()
    End Sub

    Public Sub New()
        MyBase.New(dataColumn)
    End Sub

    Public ReadOnly Property ReturnValue() As String
        Get
            Return hiddenGrid.Rows(0).Cells(0).Value.ToString()
        End Get
    End Property


    Public Function ShowDialog(initialValue As Object) As DialogResult
        Me.Expression = initialValue.ToString()
        Return MyBase.ShowDialog()
    End Function
End Class